multinode-demo/common.sh

```diff
@@ -70,7 +70,12 @@ source "$SOLANA_ROOT"/scripts/configure-metrics.sh
 SOLANA_RSYNC_CONFIG_DIR=$SOLANA_ROOT/config
 
 # Configuration that remains local
-SOLANA_CONFIG_DIR=$SOLANA_ROOT/config-local
+secondary_disk_mount_point=$(df | grep sdb | awk '{print $6}')
```

Comment:
nit: in this file we like to yell at our variables: s/secondary_disk_mount_point/SECONDARY_DISK_MOUNT_POINT/

But also this command, `df | grep sdb | awk '{print $6}'`, seems highly GCP-specific.  I think it would be better if we passed that in as an argument to `multinode-demo/validator.sh` (https://github.com/solana-labs/solana/blob/bd92f375530beb6fdc3467a3643c952884e0bf90/multinode-demo/fullnode.sh#L175-L234), and move this magic as deep as possible into `scripts/gce_provider.sh`

Reply:
or maybe `mount-additional-disk.sh` writes a symlink at /solana-config-local to the mount point?

Reply:
I was thinking that mount-additional-disk.sh could be generic if we ever wanted to make use of that feature for other disk-heavy applications, rather than linking the whole mount to the config-local, though I'm not hard set on that approach.

I have the additional disk feature locked to only GCP at the moment, but `df | grep sdb | awk '{print $6}'` should be agnostic to cloud provider, as its a standard Linux tool, and the OS increments new disks alphabetically...sda, sdb, sdc, etc.

Reply:
sda,sdb,sdc is linux-specific though, and these scripts also run on macOS.  

> I was thinking that mount-additional-disk.sh could be generic if we ever wanted to make use of that feature for other disk-heavy applications, 

Makes sense.  How about we just use well-known mount points then so no `df..awk` stuff needed - maybe `/mnt/extra-disk` or so (rather than `/solana-config-local`)